owncloudData/src/main/java/com/owncloud/android/data/user/datasources/implementation/OCLocalUserDataSource.kt

```diff
@@ -53,15 +54,19 @@ class OCLocalUserDataSource(
             UserQuota(
                 accountName = accountName,
                 available = available,
-                used = used
+                used = used,
+                total = total!!,
+                state = state!!
```

Comment:
Be careful, is this always going to be non-null? Do we only use this method in oCIS servers (where quota has `total` and `state`)?

Reply:
In oC10 `state` and `total` are null, because it doesn't use GraphAPI. Now, these parameters are nullable.